src/router/index.ts

```diff
@@ -12,6 +12,11 @@ const router = createRouter({
       name: 'home',
       component: () => import('../views/HomeView.vue'),
     },
+    {
+      path: '/planning',
```

Comment:
it is good but the problem is that component are destroyed when you change the route.
it means you lost your current location when coming back to the map, also the map tiles are reloaded.
I think it would be preferrable not to use a route but just put the planning in overlay of the home page